news-bundle/src/EventListener/DataContainer/PageListener.php

```diff
@@ -50,7 +50,7 @@ public function getAllowedArchives(): array
             ->from('tl_news_archive')
         ;
 
-        if ($user instanceof BackendUser && !$this->security->isGranted('ROLE_ADMIN')) {
+        if ($user instanceof BackendUser && !$this->security->isGranted('ROLE_ADMIN') && $user->news) {
             $qb->where($qb->expr()->in('id', $user->news));
         }
```

Comment:
Isn’t this a behavior change? If `$user->news` is empty this would now result in allowing all archives instead of none, right?

Reply:
Yeah I think it should just be

```php
$qb->where($qb->expr()->in('id', $user->news ?? []));
```

Reply:
It looks like empty `x IN ()` expressions are invalid SQL so we have to do something different here. But as this is unrelated to this pull request it should probably be done in a separate PR?
Or we just change it to `$user->news ?? []` to fix the type issue and do the rest in a separate PR?